openwisp_notifications/static/openwisp-notifications/js/notifications.js

```diff
@@ -69,30 +75,53 @@ function initNotificationDropDown($) {
     $(document).focusin(function(e){
```

Comment:
@pandafy do you think we could avoid binding on `document` here too? That `stopPropagation()` below scares me. We could incur in the same problem we had for the keyup event in other modules if we need to use the same JS event. The problem will be that to find this will probably take hours because by that time we'll have forgotten.
We can open a new issue for this.